src/models/media.ts

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { MsgType } from "../@types/event";
+
+/**
+ * @see https://spec.matrix.org/v1.7/client-server-api/#extensions-to-mroommessage-msgtypes
```

Comment:
This doesn't really document what this type represents. (Also, recommend linking to spec v1.9.)

```
Information on encrypted media attachments.

Used within `m.room.message` events that reference files, such as `m.file` and `m.image`.

@see <spec link>
```

or something?

Reply:
Likewise below, please.